x-pack/legacy/plugins/ml/server/models/job_service/new_job/categorization.ts

```diff
@@ -110,31 +123,38 @@ export function categorizationExamplesProvider(callWithRequest: callWithRequestT
     end: number,
     analyzer?: any
   ) {
-    const examples = await categorizationExamples(
+    const resp = await categorizationExamples(
       indexPatternTitle,
       query,
-      size * CATEGORY_EXAMPLES_MULTIPLIER,
+      CATEGORY_EXAMPLES_SAMPLE_SIZE,
       categorizationFieldName,
       timeField,
       start,
       end,
       analyzer
     );
 
-    const sortedExamples = examples
+    const sortedExamples = resp
       .map((e, i) => ({ ...e, origIndex: i }))
       .sort((a, b) => b.tokens.length - a.tokens.length);
-    const validExamples = sortedExamples.filter(e => e.tokens.length > 1);
+    const validExamples = sortedExamples.filter(e => e.tokens.length >= VALID_TOKEN_COUNT);
+    const sampleSize = sortedExamples.length;
+
+    const multiple = Math.floor(sampleSize / size) || sampleSize; // TODO - change this to sortedExamples.length
```

Comment:
Did you mean to do this TODO in this PR?

Reply:
it's done in this PR, i forgot to remove the comment